src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/user-settings.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "Incident intelligence user settings."
+metaDescription: "Use the incident intelligence settings to set your grace period and inactive issue time policies."
+---
+
+Incident intelligence general settings page lets you set how much time our system spends observing new activity for the purposes of correlation. It also lets you decide how much time should pass before inactive issues are closed.
+
+To find your user settings, go to **one.newrelic.com > Alerts & AI > General**.
+
+## Grace period policy [#grace-period]
+
+The grace period is the time between when an issue is created and when it's activated. For each priority level, you can set the grace period between 0 and 20 minutes. 
+
+If you've set your incident preference to `By condition and signal`, your grace period is always 0 minutes.
+
+The higher the time period, the more effectively our system analyzes new activity for correlation and to eliminate noise.
+
+Notifications are sent when the grace period time has elapsed.
+
+These settings apply to all data coming in to your account.
+
```

Comment:
Maybe it's just me, but it seems odd that each of your sentences is a paragraph.